jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -684,6 +686,64 @@ public void initializeDefaults(Attributes attributes)
                 case Deployable.TEMP_DIR -> setTempDirectory(IO.asFile(value));
                 case Deployable.CONTEXT_PATH -> setContextPath((String)value);
                 case Deployable.DEFAULT_CONTEXT_PATH -> setDefaultContextPath((String)value);
+                case Deployable.BASE_RESOURCE ->
+                {
+                    if (value == null)
+                        continue; // skip
+
+                    ResourceFactory resourceFactory = ResourceFactory.of(this);
+
+                    Resource resource = null;
+                    if (value instanceof Path path)
+                        resource = resourceFactory.newResource(path);
+                    else if (value instanceof String str)
+                        resource = resourceFactory.newResource(str);
+                    else if (value instanceof URI uri)
+                        resource = resourceFactory.newResource(uri);
+                    else if (value instanceof URL url)
+                        resource = resourceFactory.newResource(url);
+                    else if (value instanceof Resource res)
+                        resource = res;
+                    else
+                    {
+                        if (LOG.isDebugEnabled())
+                            LOG.debug("Unable to use Attribute [{}] of type [{}] to set base resource: {}",
+                                keyName, value.getClass().getName(), value);
+                        continue; // skip
```

Comment:
this should be fatal

Reply:
it is now fatal, in so far as the new `ResourceFactory.asResource(Object)` will thrown an IllegalArgumentException in this case.